dev/com.ibm.ws.st.core/src/com/ibm/ws/st/core/internal/ConsoleMonitorThread.java

```diff
@@ -60,8 +60,13 @@ public void run() {
             try {
                 int state = server.getServerState();
                 WebSphereRuntime wsRuntime = getWebSphereRuntime();
-                // server process is running, but we aren't tracking the state yet so need to add the process listener to listen for console messages
-                if (wsRuntime != null && wsServer.getUserDirectory() != null && wsRuntime.isServerStarted(wsServer.getServerInfo(), null)) {
+                int serverStatus = -2;
```

Comment:
Can you please replace this magic number with a constant? `-2` is nowhere explained.
From the `WebSphereRuntime.java` class, method `getServerStatus(…)`:
`the exit value. -1 if timeout; 0 if the server is running; 1 if the server is stopped; 2 if the status is unknown.`

Reply:
I added a comment